### PR TITLE
RFC 120: Interop Team Charter

### PR DESCRIPTION
**Pull request by foolip**
[Rendered](https://github.com/web-platform-tests/rfcs/blob/interop_charter/rfcs/interop_charter.md)

### CODE REVIEW

File: rfcs/interop_charter.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+An Interop Team is created as part of the web-platform-tests project, defined by a charter [in the team repo](https://github.com/web-platform-tests/interop-2022/pull/102). The team is tasked with maintaining [Interop 2022](./interop_2022.md), [Interop 2023](./interop_2023.md) and future such efforts.
+
+## Details
+
+The Interop 2022 [repo](https://github.com/web-platform-tests/interop-2022) and [team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will be renamed to omit the year, as described in the [Interop 2023 RFC](./interop_2023.md).
+
+The [team charter](https://github.com/web-platform-tests/interop-2022/pull/102) is made official by this RFC and substantial changes can only be made through the web-platform-tests RFC process.
+
+The Interop Team can define additional public efforts/metrics beyond Interop 2022 and 2023. The process for doing that is decided by the Interop Team, may evolve over time, and is not defined by this RFC.
```
Comment (gsnedders) [inline rfcs/interop_charter.md:13]: This kinda implies Interop could create a new BSF, which begs the question of "should Interop own BSF".

Comment (foolip) [reply rfcs/interop_charter.md:13]: Yes, I think it does imply we could create that metric, if it didn't exist already. Do you think we should explicitly give the team the authority to change or remove the BSF metric on wpt.fyi, or how do you answer the begged question?

Comment (gsnedders) [reply rfcs/interop_charter.md:13]: I don't have a particularly strong opinion, but am mildly in favour of doing so, I think.

Comment (foolip) [reply rfcs/interop_charter.md:11]: I've added this language:

> The Interop Team also maintains the "Browser Specific Failures" metric on [wpt.fyi](https://wpt.fyi/).

**Review by Ms2ger — APPROVED** [review]
wfm

**Review by jgraham — CHANGES_REQUESTED** [review]

File: rfcs/interop_charter.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+An Interop Team is created as part of the web-platform-tests project, defined by a charter [in the team repo](https://github.com/web-platform-tests/interop/pull/102). The team is tasked with maintaining [Interop 2022](./interop_2022.md), [Interop 2023](./interop_2023.md) and future such efforts.
+
+## Details
+
+The [team charter](https://github.com/web-platform-tests/interop/pull/102) is made official by this RFC and substantial changes can only be made through the web-platform-tests RFC process.
+
+The Interop Team can define additional public efforts/metrics beyond Interop 2022 and 2023. The process for doing that is decided by the Interop Team, may evolve over time, and is not defined by this RFC.
+
+The Interop Team also maintains the "Browser Specific Failures" metric on [wpt.fyi](https://wpt.fyi/).
```
[the remark this answers is not in the document]
Comment (foolip) [reply rfcs/interop_charter.md:13]: Turning that into a line suggestion:

```suggestion
The Interop team will take ownership of the existing Browser Specific Failures metric, and manage it according to the rules in its charter.
```

**Review by jgraham — APPROVED** [review]

File: rfcs/interop_charter.md (hunk 1 above)
Comment (foolip) [reply rfcs/interop_charter.md:11]: It's also mentioned in https://github.com/web-platform-tests/interop/pull/102 now.

**Comment by foolip** [comment]
This is on the agenda again in https://github.com/web-platform-tests/interop/issues/329, and if we have consensus in that meeting I'll go ahead and land this RFC. It's long had enough approvals from the wpt RFC process perspective, just waiting on https://github.com/web-platform-tests/interop/pull/102 to be finished.

**Comment by foolip** [comment]
I'd like to merge this now. @gsnedders you haven't reviewed yet, would you like to do that?

**Comment by foolip** [comment]
Per the RFC process this can be merged, we were just waiting for https://github.com/web-platform-tests/interop/pull/102 all this time. I'll merge it now. @gsnedders was very much involved in the review of https://github.com/web-platform-tests/interop/pull/102, I just wanted to check one final time everyone is happy.

**Pull request merged**